### PR TITLE
OSM Machines

### LINKED ISSUE #84: check OSM machines
We should try to get the penny machines that are tagged on OSM (tag: elongated coins, category: verkaufsstelle) and compare the coordinates to our database.

### PR DESCRIPTION
**Pull request by jannisborn**
Adding ~450 new machines! Remaining 400 ones from OSM were processed and identified as already existing in our DB.

This PR will close #84 

Some steps completed towards automatic OSM processing (#201 ) but more work required

### CODE REVIEW

File: backend/pennyme/osm.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        geojson = {
+            "type": "Feature",
+            "geometry": {
+                "type": "Point",
+                "coordinates": [float(machine.lon), float(machine.lat)],
+            },
+            "properties": {
+                "name": title,
+                "active": True,
+                "area": country,
```
Comment (NinaWie) [inline backend/pennyme/osm.py:91]: shouldn't this be "area": match
since you want to use the area from our list that was matched by fuzzysearch

**Review by NinaWie — APPROVED** [review]
Very cool! 
One question: If we run the location differ now, and there is a new machine that matches with one of the elongated-coin machines, then the external_url would be replaced with the one from the pennycollector website, right? Because the elongated-coin machines are basically treated like the link = null machines?

Also, you should send me your vscode formatting settings; we should use the same because the format diff is very annoying.

**Comment by jannisborn** [comment]
@NinaWie exactly this was the idea to have the pennylocator website as the main data source. So if it's available it's preferred over elongated-coin website

As for formatting, let's do this in a separate PR for all the backend code and set up a small CI routine, or even a pre-commit hoook with RUFF

File: backend/pennyme/osm.py (hunk 1 above)
Comment (jannisborn) [reply backend/pennyme/osm.py:91]: Good catch!!!!!

**Pull request merged**